README.md

```diff
@@ -19,9 +19,8 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
 has a minimalist design. It excludes unnecessary devices and guest-facing
 functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
-hardware utilization. Firecracker currently supports Intel CPUs, with planned
-AMD and Arm support. Firecracker will also be integrated with popular container
-runtimes.
+hardware utilization. Firecracker currently supports Intel, AMD and Arm CPUs. 
+Firecracker will also be integrated with popular container runtimes.
```

Comment:
There are some container runtimes which are currently integrated with Firecracker. I d rephrase this sentence for accuracy reasons to something like: "Firecracker can also be integrated in container runtimes. Checkout [Kata Containers with Firecracker](https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support)".